doc/faq/howto_faq.rst

```diff
@@ -106,13 +106,12 @@ Save transparent figures
 The :meth:`~matplotlib.pyplot.savefig` command has a keyword argument
 *transparent* which, if 'True', will make the figure and axes
 backgrounds transparent when saving, but will not affect the displayed
-image on the screen.
+the image on the screen.
```

Comment:
```suggestion 
image on the screen.
```

Reply:
I think these errors are due to various English standards.

Reply:
I don't think there is any version of English where inserting an extra determinant between an adjective and its noun is considered correct.

Reply:
"... will not affect the displayed image on the screen."
versus
"... will not affect the displayed the image on the screen."

The former is correct. Perhaps the split line is causing confusion?